bitswap/network/httpnet/httpnet.go

```diff
@@ -508,7 +513,7 @@ func (ht *Network) Connect(ctx context.Context, pi peer.AddrInfo) error {
 		err = ht.connectToURL(ctx, pi.ID, u, "GET")
 		if err != nil {
 			errs = append(errs, fmt.Errorf("%s: %s", u.Multiaddress.String(), err))
```

Comment:
Note: You could also use multierr.Append here.
```suggestion
			errs = multierr.Append(errs, fmt.Errorf("%s: %s", u.Multiaddress.String(), err))
```
Then `errs` would be declared as :
```
errs := ErrNoSuccess
```

Reply:
will fix next time I touch this code

Reply:
ah wait, I think it is done this way because multierr.Append prints errors in a different way than multierr.Combine. The former joins them in a single line, and the latter nicely formats them in multiple lines. (IIRC!)